google-cloud-bigtable/lib/google/cloud/bigtable/chunk_reader.rb

```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+module Google
+  module Cloud
+    module Bigtable
+      # Invalid row state error
+      class InvalidRowStateError < Google::Cloud::Error
+        attr_reader :data
+
+        def initialize message, data = nil
+          super(message)
+          @data = data if data
+        end
+      end
+
+      class ChunkReader # :nodoc:
```

Comment:
You should mark this class with YARD's `@private` to indicate it should not be part of the public API. We don't use RDoc annotation such as `:nodoc:`.

Reply:
Go it.